src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const objectWithMethods = {
+    // Basic math operations, it MIGHT BE done, but probably not.
+    add: function (number) {
+      this.result += number;
+    },
```

Comment:
You are using the `function` keyword for methods in an object. This is not wrong, but it's more modern and concise to use the shorthand method syntax. This applies to all methods in your object. [CODE KNOWLEDGE]